src/components/PageHeading.js

```diff
@@ -1,22 +1,76 @@
-import { Component } from 'react';
-import { div, hh, img, h2, span } from 'react-hyperscript-helpers';
-import './PageHeading.css';
+import {Component} from 'react';
+import {div, hh, img, h2, span} from 'react-hyperscript-helpers';
 
 export const PageHeading = hh(class PageHeading extends Component {
```

Comment:
Outside the scope of this PR, but these components can easily be pure functions instead. This shouldn't hold up the PR, but if you want an example, see the work done [in the DAR pages](https://github.com/DataBiosphere/duos-ui/blob/develop/src/pages/dar_application/DataAccessRequest.js#L38)